frameworks/digital-outcomes-and-specialists-5/questions/declaration/safeguardOfficialInformation.yml

```diff
@@ -1,4 +1,5 @@
 name: Safeguarding official information
+id: safeguardOfficialGovernmentInformation
```

Comment:
Why not `safeguardingOfficialInformation`?

Reply:
The purpose of this change is to change the ID of this question so that it is not pre-filled.

`safeguardingOfficialInformation` is the current ID as generated from the file name (AFAIK). So we need to set the ID to something different in order to avoid prefill.